src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -41,8 +41,16 @@ public void RelativeSearchPath_Is_Null()
         [Fact]
         public void TargetFrameworkTest()
         {
-            // On Uap we use the Microsoft.DotNet.XUnitRunnerUap instead of the RemoteExecutorConsoleApp
-            string targetFrameworkName = PlatformDetection.IsUap ? ".NETCore,Version=v5.0" : "DUMMY-TFA";
+            string targetFrameworkName = "DUMMY-TFA";
+            if (PlatformDetection.IsInAppContainer)
+            {
+                targetFrameworkName = ".NETCore,Version=v5.0";
+            }
+            if (PlatformDetection.IsNetNative)
```

Comment:
Nit: else if?

Reply:
I'll fix in next PR

Reply:
Ok

Reply:
The if is correct. If you're inside an app container but also use NET Native (no existing configuration in corefx today) you need the latter TFM. Of course the if blocks could be reversed and combined to an else if but as this is just test code, I think the current form is ok

Reply:
> I think the current form is ok

I think it's confusing because you end up stomping over the value that was just written if both conditions are true.  It'd be better to reverse the order and make them an if/else if.

Reply:
I'm fine with that.